frontend/src/store/objectStore.ts

```diff
@@ -73,15 +73,17 @@ export const useObjectStore = defineStore('objectStore', () => {
         const uniqueIds = [...new Set(permResponse.map((x: { objectId: string }) => x.objectId))];
         const objects = (await objectService.listObjects({ objId: uniqueIds, ...params })).data;
         const metadataResponse = (await objectService.getMetadata(null, { objId: uniqueIds })).data;
+        const objectTaggingResponse = (await objectService.getObjectTagging()).data;
 
         objects.map(async (obj: any) => {
           const metadata = metadataResponse.find((x: Metadata) => x.objectId === obj.id);
-          // TODO: Tags
 
           if (metadata) {
             obj.metadata = metadata;
             obj.name = metadata.metadata.find((x: { key: string }) => x.key === 'name')?.value;
           }
+
+          obj.tag = objectTaggingResponse.find((x: ObjectTag) => x.objectId === obj.id);
```

Comment:
Receiving 422 when trying to get tags

Reply:
Might be related to some incoming changes on COMS that haven't been folded in yet.

Reply:
/object/tagging should be available in COMS dev now